### PR TITLE
Ember post

### PR DESCRIPTION
**Pull request by sbatson5**
## What's in this PR?

Initial draft of my ember post which tries to summarize the fact that I love ember but it's lack of marketshare can make it hard to find a job.

### CODE REVIEW

File: content/posts/2019-05-21---Working-In-Ember.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+description: "After years of working in Ember, I recently took a job working in other technologies. I thought I'd take a moment to reflect on my time with the framework."
+---
+
+![Laptop and notepad](https://i.imgur.com/bM8ljia.jpg)
+<aside>Photo by Lukas from Pexels</aside>
+
+I worked with Ember.js for several years, across multiple companies.
+In fact, it was the first JavaScript framework I had worked in professionally.
+I am so grateful for my experience with the community and the learning curve -- I am a better developer because of the conventions it forced on me.
+But after 4+ years of writing Ember, I took another job where I won't be writing it anymore, and I have a few thoughts overall on the framework.
```
Comment (eshtadc) [inline content/posts/2019-05-21---Working-In-Ember.md]: I think the overall of this post is a post of reflection as you are closing a chapter in your life with Ember. Maybe emphasize the point that this is "a few thoughts" and it makes the rest of the post have more context.

File: content/posts/2019-05-21---Working-In-Ember.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    <footer>
+			<cite>— Overly defensive web devs.</cite>
+		</footer>
+	</blockquote>
+</figure>
+
+I feel like I see a ton of tweets and posts from really smart people in the community answering the question "is Ember dead?" with a resounding "heck no!"
+But honestly, I don't actually see a lot of people asking that question (except for the occasional troll).
+Nobody thinks Ember is going to disappear -- there are a lot of companies that are invested in it, a community that is passionate about maintaining it and enough smart people behind it to help drive it into the future.
+Nobody seriously thinks it is going to just end and disappear, so we need to stop being overly defensive about this.
```
Comment (eshtadc) [inline content/posts/2019-05-21---Working-In-Ember.md]: A little repetitive with the "nobody thinks ... disappear" twice but that may be intentional.

Totally agree with you on this point, by the way. It's kind of the "me thinks thou doth protest too much" type of thing.

File: content/posts/2019-05-21---Working-In-Ember.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+Secondly, I formed a lot of good habits because of Ember in terms of structuring an application, writing tests, handling how data moves through my components and more.
+But I also formed some bad habits without realizing it.
+I didn't quite know how to write native JavaScript classes.
+Things like `Component.extend({})` over `Foo extends Component`, `init` over `constructor`, `this._super` or `super()` and more, made me feel like I had fallen behind other JavaScript developers.
+
+<figure>
+	<blockquote>
+		<p>But Ember paved the way! Those things weren't around when Ember started!</p>
+    <footer>
+			<cite>— People who miss the point.</cite>
```
Comment (eshtadc) [inline content/posts/2019-05-21---Working-In-Ember.md]: I love your citations :)

File: content/posts/2019-05-21---Working-In-Ember.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+	<blockquote>
+		<p>Just because you can explain why something is bad, doesn't mean it's not bad</p>
+    <footer>
+			<cite>— Me.</cite>
+		</footer>
+	</blockquote>
+</figure>
+
+Even if your current company writes Ember and you love your job and you never want to leave, there may come a time when that isn't the case.
+Wouldn't you want to feel like you could find another job quickly if you had to?
```
Comment (eshtadc) [inline content/posts/2019-05-21---Working-In-Ember.md]: There is a case to be made for being a better developer in any framework from learning more than one.

**Comment by tschoartschi** [comment]
I really like your blog-post but I'm not 100% sure if it is a good fit for the Ember 2019 Roadmap. I think the #EmberJS2019 blog-posts should suggest how the framework can evolve and which features should be implemented. 

Since you seem to have extensive knowledge in Ember and other eco-systems it would be great to read where Ember is behind and where Ember has to improve.

When I read your blog-post I would think that you suggest mainly to improve marketing to increase adoption and to stick to industry-wide tools like webpack. I think the feedback from people like you is very helpful for the Ember community because you know the in's and out's of the other tools as well. 

This can help to implement one claim of the Ember maintainers: 

**Eventually all the good ideas will end up in Ember** 

**tldr;** would be great to read more about specific suggestions where and how to improve :-)

**Comment by sbatson5** [comment]
Thank you so much for reading/review @tschoartschi!

I was actually thinking the same thing -- that this needs to be pointed more toward where I'd like to see Ember end up in the near future, based on my experiences. I've just been buried at work and haven't had a chance to revise this post yet.

File: content/posts/2019-05-21---Working-In-Ember.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+And yes, most companies won't disqualify you for not know their tech stack... but what about the ones that will?
+It would stink to miss out because you couldn't write a class properly in JavaScript or explain how to do something webpack.
```
Comment (gwak) [inline content/posts/2019-05-21---Working-In-Ember.md]: > how to do something **in** webpack

File: content/posts/2019-05-21---Working-In-Ember.md [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+I think of testing in terms of `unit`, `integration` and `acceptance` and understand when each makes sense.
+I know how to give back to a community through open source documentation or tutorials -- the Ember community taught me these skills!
+
+# Conclusion?
+
+I feel like I mostly complained in this post, but I hope it doesn't come across that way.
+I love Ember and would be happy writing it for the rest of my career (I honestly feel this way).
+But it didn't seem like that was up to me -- I'm not in a position to make the call on what technology my company uses.
+Maybe one day I'll be in that position, but if the landscape stays the same, I'd probably go with React simply because so many more people know it.
+My hope is that Ember keeps leaning into the conventions that the rest of the JavaScript world have adopted ([Ember Octane a great step](https://emberjs.com/editions/octane/)) and that those with experience in Vue or React can jump into an Ember project and not feel lost.
```
Comment (gwak) [inline content/posts/2019-05-21---Working-In-Ember.md]: > ([Ember Octane **is** a great step](https://emberjs.com/editions/octane/))
